ci/playbooks/content_provider/pre.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: all
+  tasks:
+    - name: Clone repos in the job workspace
+      include_role:
+        name: prepare-workspace
+
+    - name: Install ansible-core
```

Comment:
I think the test_deps role should cover that "better".

Reply:
test_deps role is not used in the content provider job. Will I include test_deps role in favor of repo-setup?

Reply:
or maybe that's to set in the ci_setup actually. For now that role only creates directories, but it may also ensures some basic packages are present (though... ansible-core should be there to run the playbook, right? ;))

Let's keep it as-is, we may iterate later if there's an actual need.